### PR TITLE
Update locale modal images and content

### PR DESCRIPTION
**Pull request by RichKummer**
### Related Ticket(s)

#6589

### Description

Update to `Locale modal` page to account for visual updates to bring the examples up-to-date. Added additional content to explain the different states of the modal.

### Changelog

**New**

- Two new images with captions.
- Additional paragraph on second state of modal.

**Changed**

- Slight changes to copy in introductory section.

**Removed**

- Deleted original, out-of-date modal image

### CODE REVIEW

File: src/pages/components/localemodal.mdx [hunk 1]
```diff
@@ -21,8 +21,26 @@ import ResourceLinks from "components/ResourceLinks";
...
+The user can click or tap a region to view a searchable list of locations and languages. They can select an option to reload the page as the corresponding location and language.
+
+<Row>
+<Column colMd={8} colLg={8}>
+
+![Default](../../images/component/locale-modal/locale-modal-country.png)
+
+</Column>
+</Row>
+<Caption>The second state of the modal displays a searchable list of location and region pairings.</Caption>
```
Comment (janchild) [inline src/pages/components/localemodal.mdx:44]: @RichKummer, should this be `location and language pairings`?

**Review by janchild — COMMENTED** [review]
This looks good, Rich. Just one question about the second caption.

File: src/pages/components/localemodal.mdx (hunk 1 above)
Comment (RichKummer) [reply src/pages/components/localemodal.mdx]: It should be, region is the first state. Just pushed an update with the new language.

**Review by janchild — APPROVED** [review]
Looks good!

**Pull request merged**